src/Sarif.FunctionalTests/Properties/AssemblyInfo.cs

```diff
@@ -1,8 +1,5 @@
-﻿// /********************************************************
-// *                                                       *
-// *   Copyright (C) Microsoft. All rights reserved.       *
-// *                                                       *
-// ********************************************************/
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
I just noticed this when I happened to choose this file to copy into the new ValidationTests project. It's not like I was going around looking for it.